manifest.json

```diff
@@ -36,6 +36,8 @@
         "popup.js",
         "shadertoy-plugin.js",
         "shadertoy-plugin-profile.js",
-        "shadertoy-plugin-home.js"
+        "shadertoy-plugin-home.js",
+        "/js/jszip-3.1.5.js",

```

Comment:
please move additional files to "lib" folder

/lib/jszip-3.1.5.js
/lib/node-sanitize-filename.js

there are js files in root folder so adding "js" folder for only 2 new files is not good idea IMO